src/devices/Is31fl3731/samples/MatrixRgb/Program.cs

```diff
@@ -80,7 +80,7 @@ void TestPixels(byte r, byte g, byte b)
         foreach (int x in Enumerable.Range(0, 5))
         {
             matrix.Fill(0);  // Clear display
-            Color color = Color.FromRgb(r, g, b);
+            Color color = Color.FromArgb(r, g, b);
```

Comment:
this is confusing since argb refers to 4 values but we're passing 3, why aren't we using FromRgb instead?

Reply:
Nvm, looked at the wrong Color class documentation